cms/toolbar/utils.py

```diff
@@ -117,14 +120,41 @@ def get_toolbar_from_request(request):
     return getattr(request, 'toolbar', EmptyToolbar(request))
 
 
+def get_querystring_modifier(obj, url, language=None):
```

Comment:
Now that this has been simplified, I am not sure the name of this function is reflective of what it does - it doesnt really _get_ anything, it _adds_ the live url to the querystring. So to me something like `add_live_url_to_querystring` would be more descriptive.